packages/component-library/components/Data/Data.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from "classnames"
```

Comment:
IMO this component got _pretty close_. Outstanding items in the React component: 
* Tests
* Accessibility checks 
* Explore all use cases +, -, $, %, different currencies, etc etc